README.md

```diff
@@ -1,15 +1,20 @@
 ![node-js](assets/nodejs-black.png) Help
 ====
 
-Need help with Node.js? You've come to the right place! [Create a new issue](https://github.com/nodejs/help/issues/new)!
+Need help with Node.js? You've come to the right place! 
 
-Join the conversation!
+## [Create a new issue](https://github.com/nodejs/help/issues/new)
 
-[![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/nodejs/help)
+If you have a question about Node.js, [create a new issue](https://github.com/nodejs/help/issues/new)
+and ask there!
 
-## Please feel free to ask
+## Chat Resources
 
-[Please click here](https://github.com/nodejs/help/issues/new) to create a new issue if you have any questions. :smile:
+If you would prefer to get help via live chat rather than the issue tracker in
+this repository, you can try:
+
+* [Freenode #node.js channel](https://webchat.freenode.net/?channels=node.js&uio=d4)
+* [Gitter Node.js room](https://gitter.im/nodejs/help)
```

Comment:
Argh! I meant to link to the same Gitter room as is [in the nodejs/node repo README](https://github.com/nodejs/node#resources-for-newcomers) but apparently, I messed up. Let me update....